File: django_src/apps/register/urls.py
from django.urls import path
from .views import MainView, SelectCarreraView, SelectCarrerSpecialization, SelecThemeView
from .complete_profile_view import complete_profile_view
from .add_mentor_exp_view import add_mentor_exp_view
from .views import register_sucess_view
app_name="register"

urlpatterns = [
    path("select_carrera/", SelectCarreraView.as_view(), name="select_carrera"),
    path("select_carreer_specialization/<str:name>", SelectCarrerSpecialization.as_view(), name="select_specialization"),
    path("select_themes/<str:name>", SelecThemeView.as_view(), name="select_themes"),
    path("complete_profile/", complete_profile_view, name="complete_profile"),
    path("add_exp/", add_mentor_exp_view, name="add_exp"),
    path("success/", register_sucess_view, name="success"),
    # Todo
    # path("mentors/<int:pk>",mentor_detail_view, name="mentor_detail")
    path("", MainView.as_view(), name="index"),

]

File: static/src/js/config/urls.ts
export const base = "/register"
export const registerUrls = {
    complete_profile: (carreer: string, profile: string, no_spec: boolean = false) => {
        const params = new URLSearchParams("")
        params.append("carreer", carreer)
        params.append("profile", profile)
        if (no_spec) params.append("no_spec", "")
        return `/register/complete_profile?${params.toString()}`
    },
    select_specialization: (carreer: string) => {
        return `/register/select_carreer_specialization/${carreer}`
    },
    select_themes: (carreer: string) => {
        return `/register/select_themes/${carreer}`
    },
    add_exp: `${base}/add_exp`
}
